Automattic/wp-calypso
# feat(TrackingLink): update URL generators

#### Changes proposed in this Pull Request

- Update tracking link generators to add DHL/UPS
- Ensure that if carrier is invalid, function doesn't break

This is similar to https://github.com/Automattic/woocommerce-services/blob/develop/client/extensions/woocommerce/woocommerce-services/views/shipping-label/tracking-link.js

#### Testing instructions



- Ensure the site is public (you can use ngrok to forward the domain)
- Ensure the site has DHL enabled (in the MC site tools, it should have "can use dhl express" enabled)
- Create a non-domestic order as a regular customer
- Go to the order's page at `http://calypso.localhost:3000/store/order/my-store.ngrok.io`
- Purchase a DHL label
- Tracking number should be working correctly

Fixes https://github.com/Automattic/woocommerce-shipping-issues/issues/109
